### ISSUE TITLE
Incompatible peer dependency to "@angular/core" (requires "^9.1.0")

### ISSUE THREAD

**Issue by MortenGregersen** [sneas/ionic-native-http-connection-backend#136]
### Incompatible peer dependency to "@angular/core" (requires "^9.1.0" (extended), would install "11.2.3")
When we want to update to Angular 11 we get the following error. Could this be updated, to enable Angular 11?

```
$ npx ng update @angular/cli
The installed local Angular CLI version is older than the latest stable version.
Installing a temporary version to perform the update.
Installing packages for tooling via npm.
Installed packages for tooling via npm.
Using package manager: 'npm'
Collecting installed dependencies...
Found 131 dependencies.
Fetching dependency metadata from registry...
                  Package "ionic-native-http-connection-backend" has an incompatible peer dependency to "@angular/core" (requires "^9.1.0" (extended), would install "11.2.3").
✖ Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "/private/var/folders/mk/vcl_6fk16mx6fkm290g7knd80000gn/T/ng-nCpvGJ/angular-errors.log" for further details.
```

### Environment
* angular: 10.2.0
* ionic-native-http-connection-backend: 8.1.2
* @ionic/angular: 5.5.2
* @ionic-native/http: 5.28.0

### Steps to reproduce
`npx ng update @angular/cli`

**Comment by sneas** [comment]
Hi Morten, I've updated the library's `peerDependencies`. Now it accepts all the Angular versions which are >= 9.1.0. The new version `8.1.3` has been released. Cheers!

**Comment by MortenGregersen** [comment]
Awesome! 🎉 Thank you @sneas!

**Comment by MortenGregersen** [comment]
What is the timing for when a version is moved to `latest` instead of `next`? I see that 8.1.3 is still `next`.

**Comment by sneas** [comment]
Hi Morten, I've just released `8.1.3` under the `latest` tag.

Tbh the whole idea with `latest` and `next` tags on this project turned out quite misleading for users (and for myself). There is no well-defined strategy for moving versions from `next` to `latest`, thus I'm going to eliminate `next` tag in the nearest future.

**Comment by eggbeard** [comment]
Is there a requirement for `"@ionic-native/http": "~5.12.0"` rather than `"@ionic-native/http": "^5.12.0"`

Seems odd to have a different peerDependency than devDependency
